scripts/utils/fixEntitiesDonationCounter.js

```diff
@@ -172,10 +188,10 @@ const updateEntity = async (model, type) => {
         donationCounters.forEach(dc => {
           const oldDC = oldDonationCounters.find(item => item.symbol === dc.symbol);
           if (
-            oldDC === undefined ||
```

Comment:
This checking was wrong because if oldDc will be undefined so goes to if statement there will throw exception

Reply:
You are right partially... it was wrong and we should put guard in the below statement. But the condition wouldn't be true of `oldDC === undefined` and consequently `shouldUpdateEntry` will not be true.
Please revert it but put guard on accessing oldDC in the statement

Reply:
done